src/lib/Behat/Component/UniversalDiscoveryWidget.php

```diff
@@ -116,6 +111,45 @@ static function (ElementInterface $element) {
         );
     }
 
+    public function bookmarkContentItem(): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('bookmarkButton'))->click();
+        $this->getHTMLPage()->setTimeout(3)->waitUntil(function () {
+            return $this->isBookmarked();
+        }, 'The icon did not change to bookmarked one');
+    }
+
+    public function isBookmarked(): bool
+    {
+        $htmlFragment = $this->getHTMLPage()
+            ->find($this->getLocator('bookmarkButton'))
+            ->getOuterHtml();
+
+        return strpos($htmlFragment, 'bookmark-active') !== false;
+    }
+
+    public function changeTab($tabName): void
+    {
+        $tabLocator = new CSSLocator('', sprintf($this->getLocator('categoryTabSelector')->getSelector(), $tabName));
+        $this->getHTMLPage()->find($tabLocator)->click();
+        $selectedTabLocator = new CSSLocator('', sprintf($this->getLocator('selectedTab')->getSelector(), $tabName));
+        $this->getHTMLPage()->find($selectedTabLocator)->assert()->isVisible();
```

Comment:
I'd go with `$this->getHTMLPage()->find($selectedTabLocator)->assert()->textEquals($tabName)` here